src/lib/interactionManager/flow.ts

```diff
@@ -15,7 +15,7 @@ export abstract class Flow {
   abstract async handleInteractionToken(
     token: JWTEncodable,
     messageType: InteractionType,
-  ): Promise<void>
+  ): Promise<boolean>
```

Comment:
indicates if the state change caused by handling this token is valid or invalid. This may be restrictive for flows which can branch or have multiple valid next states to choose from, but for our current flows (which are linear) it works. If this needs to be changed to something more descriptive then it can be